### ISSUE TITLE
Possible wrong documentation about pylint

### ISSUE THREAD

**Issue by demestav** [pydanny/cookiecutter-django#2744]
## What happened?
In linters documentation [1], it is stated that 

> [pylint] is included in flake8's checks

. 

However, running `flake8 <random_python_file>` in one of my projects, shows no errors, while running `pylint <the_same_random_python_file>` shows some warnings.

## What should've happened instead?
If I interpret the documentation [1] correctly, then pylint's warnings should show when running flake8. (I am not aware if this is possible, anyway).

[1] https://github.com/pydanny/cookiecutter-django/blob/55ef75e7a03c272c6ee712514368745a6ca07523/docs/linters.rst#pylint

**Comment by browniebroke** [comment]
That looks wrong indeed, I don't see anything wiring `flake8` and `pylint` together. Looks like this section was mostly written 5 years ago, so things have changed since then.

Things I could think of:

- We've removed a flake8 plugin: I didn't find any `flake8` plugin for `pylint` so I don't think it's possible.
- Rules in `pylint` were all in `flake8` _at the time_, but now `pylint` added more rules which are not in `flake8`?

Or maybe this statement was always wrong and we missed this...

From a personal perspective, I'm not super familiar with `pylint` these days, in my distant memories I think it was too picky with errors for my taste, so I've slowly moved away and sticked to `flake8` only. I don't know if this past experience is still accurate though...

So in summary, I see 2 ways of fixing this:

1. Remove `pylint` if we think it doesn't bring enough value
2. Fix this statement in the documentation and update the CI setup to run `pylint`

Let me know if I miss anything

**Comment by demestav** [comment]
I agree. Since this issue is about the documentation, we could go with option 2. I would submit a pull request, but I am not familiar with the CI setup yet. Would that be adding pylint to **tox.ini** file?

**Comment by luzfcb** [comment]
> From a personal perspective, I'm not super familiar with pylint these days, in my distant memories I think it was too picky with errors for my taste, so I've slowly moved away and sticked to flake8 only. I don't know if this past experience is still accurate though...

I have the exact same feelings about pylint.

https://github.com/vintasoftware/python-linters-and-code-analysis
